docs/managed-datahub/operator-guide/setting-up-remote-ingestion-executor.md

```diff
@@ -13,7 +13,7 @@ import FeatureAvailability from '@site/src/components/FeatureAvailability';
 
 :::note
 
-The Remote Executor can now also be used for monitoring of ingestion sources.
+The Remote Executor is only available for DataHub Cloud. If you are looking to use Observability features in Acryl DataHub Cloud, remote executor must be enabled in your Acryl instance. Please contact Acryl support to enable Remote Executor in your instance.
```

Comment:
I expected to see a new section detailing what to do if you have V1 executor to upgrade to V2. The customer has to deploy the new images etc right? 

"Upgrading to Remote Executor V2"

Reply:
My understanding from the conversation with Santhosh was that we want a not directing customers to Acryl Support for cases when they want to use Observability, but their instance has not been enabled for this. Customer support can then provide all the guidance on upgrade because in either case, the upgrade has to be done in sync with Acryl.